Gulpfile.js

```diff
@@ -40,6 +40,29 @@ var UI_STYLES_FILE_PATH = '/client/ui';
 var UI_STYLES_TEMP_DIR  = path.join('lib', UI_STYLES_FILE_PATH);
 
 
+function promisify (fn) {
```

Comment:
Let's use https://www.npmjs.com/package/es6-promisify then. This covers both cases discussed here. We need to replace it in hammerhead/browser-natives as well for consistency, please fill the issue where.